### DISCUSSION TITLE
ZiGate Ethernet (wired) Gateway / Bridge open source HW based on ESP32

### DISCUSSION THREAD

**Discussion by Hedda** [arendst/Tasmota#12902] [Ideas]
Tip: ZiGate-Ethernet based on ESP32 has just been released as an open source hardware by Frédéric Dubois a.k.a. fairecasoimeme

![image](https://user-images.githubusercontent.com/6320001/129525575-05dbac49-6782-4769-b31c-61e3a7113671.png)

Seems like Tasmota32 compatible has partially been tested as they have some documentation for it already:

https://zigate.fr/documentation/installation-de-tasmota-pour-la-zigate-ethernet/

It features a partial (16-pin) Raspberry Pi compatible GPIO header and is designed to be used as Ethernet to UART/serial radio gateway.

Obviously, it can be also be used as an Ethernet to Bluetooth Low Energy (BLE) gateway or generic Ethernet to serial GPIO proxy.

"Expansion – 16-pin GPIO header with up to 10x GPIO, 7x ADC, UART, 5V, 3.3V, and GND"

![image](https://user-images.githubusercontent.com/6320001/129688891-0ba63ff2-8c83-43ef-bdbc-8b2d67b47909.png)

Looks to features an ESP32-WROOM-32E with ESP32 dual-core processor and 16MB flash. LAN872 enable 10/100M Ethernet RJ45 port.

https://github.com/fairecasoimeme/ZiGate-Ethernet

https://zigate.fr/produit/zigate-ethernet/

https://zigate.fr/documentation/descriptif-de-la-zigate-ethernet/

https://zigate.fr/documentation/installation-de-tasmota-pour-la-zigate-ethernet/

Note! This board ships without Zigbee radio PCB board shield and antenna as standard so you would need to add that to use it:

ZiGate officially supports it with their "[PiZiGate](https://zigate.fr/produit/pizigatev2/)" radio but with third-party ESP32 firmware like Tasmota it could in theory support other radio boards as either only a Serial-to-IP proxy server functionality (for [PiZiGate](https://zigate.fr/produit/pizigatev2/) and RaspBee) or native Tasmota ZBridge coordinator gateway/bridge to MQTT if use a Texas Instruments or Silicon Labs compatible radio board and Zigbee stack firmware.

https://elelabs.com/products/elelabs-zigbee-shield.html

https://www.popp.eu/zb-shield/

https://zig-star.com/projects/zigbee-shield/

https://www.tindie.com/products/GiovanniCas/zigbee-hat-with-cc2538-for-raspberry/

https://github.com/egony/cc2652p_E72-2G4M20S1E/wiki/Home-EN

https://github.com/egony/cc2652p_cc1352p_RF-STAR/wiki/Home-EN

https://phoscon.de/en/raspbee

https://zigate.fr/produit/pizigatev2/

![image](https://user-images.githubusercontent.com/6320001/129526590-036298d5-b689-417c-9159-bd2ed99b7202.png)

![image](https://user-images.githubusercontent.com/6320001/129687507-aa255542-d287-41cc-95b8-e1b87cd175a5.png)

The board is apparently sized to fit the enclosure case made for Raspberry Pi 4

![image](https://user-images.githubusercontent.com/6320001/129525814-bff49b63-1b7a-49ff-852d-9a667875a257.png)

The serial-over-IP function should ex. work out-of-the-box with Home Assistant's ZHA integration via the radio libraries for zigpy

https://github.com/zigpy/zigpy-zigate

PS: According to fairecasoimeme it has also been tested with ESPHome using esphome-zbbridge as an alternative firmware:

https://github.com/thegroove/esphome-zbbridge

https://github.com/fairecasoimeme/ZiGate/issues/366#issuecomment-898576244

**Comment by s-hadinger** [comment]
Interesting and thanks for sharing. Instructions to load Tasmota on it are already there. If the zigbee module are TI or EZSP based, they should work.

**Reply by Hedda** [reply]
> Instructions to load Tasmota on it are already there. If the zigbee module are TI or EZSP based, they should work.

PiZiGate and RaspBee are not, but Elelabs/Popp are EFR32MG12 with EZSP, and ZigStar/Egony/GiovanniCas are TI CC2652P or CC2538

https://elelabs.com/products/elelabs-zigbee-shield.html <- Silabs EFR32MG12 radio with EZSP serial interface

https://www.popp.eu/zb-shield/ <- Silabs EFR32MG12 radio with EZSP serial interface

https://zig-star.com/projects/zigbee-shield/ <- Texas Instruments CC2652P radio with TI serial interface

https://github.com/egony/cc2652p_E72-2G4M20S1E/wiki/Home-EN  <- Texas Instruments CC2652P radio with TI serial interface

https://github.com/egony/cc2652p_cc1352p_RF-STAR/wiki/Home-EN  <- Texas Instruments CC2652P radio with TI serial interface

https://www.tindie.com/products/GiovanniCas/zigbee-hat-with-cc2538-for-raspberry/   <- TI CC2538 radio with TI serial interface

https://phoscon.de/en/raspbee  <- Desden Elektronik RaspBee II (RaspBee II ) radio with deconz serial interface

https://zigate.fr/produit/pizigatev2/  <- PiZiGate+ use NXP NXP Jennic JN516x radio with zigate firmware and zigate serial interface
